### ISSUE TITLE
Issue with downloading larger files on Chrome specifically

### ISSUE THREAD

**Issue by asynch8** [axios/axios#6358]
### Describe the bug

I'm fetching a files using **Axios 1.6.8** and Chrome, and when I try to fetch a 'larger'(16mb) zip-file within Chrome, it fails saying network error. 
The backend responds 200, both according to the backend itself and to the browser, and even Axios agrees, shown by this error:
```
net::ERR_FAILED 200 (OK)
```

The rest of the stacktrace contains nothing really interesting.

Switching to Firefox fixes the issue, and so does replacing Axios with native fetch.

I've also tried it in Vue and React separately, so probably not a framework issue.

### To Reproduce

https://codesandbox.io/p/sandbox/nice-tristan-vqcwnm
I've provided some description and ways to reproduce in this sandbox.

### Code snippet

```js
Code I was running:
 
axios.get(url, { responseType: 'blob', headers: {
        ...
      } })
  .then(async response => {
    let blob = new Blob([response.data], { type: contentType })
    
    const link = document.createElement('a')
    link.href = URL.createObjectURL(blob)
    console.log({ name })
    link.download = `${name}${contentType === 'text/directory' ? '.zip' : ''}`
    link.click()
    URL.revokeObjectURL(link.href)
  }).catch((error) => {
    console.error(error);
  })
 ````
 
 And here's the code I replaced it with:
 ```
 fetch(url, {
    headers: {
      ...
    }
  })
  .then(response => response.blob())
  .then(async blob => {
    blob = new Blob([blob], { type: contentType })
    
    const link = document.createElement('a')
    link.href = URL.createObjectURL(blob)
    link.download = name;
    link.click()
    URL.revokeObjectURL(link.href)
  }).catch((error) => {
    console.error(error);
  })
  ```
```


### Expected behavior

The expected behavior is for Axios to successfully download my file and not throw an internal error.

### Axios Version

1.6.8

### Adapter Version

_No response_

### Browser

Chrome

### Browser Version

123.0.6312.123

### Node.js Version

_No response_

### OS

Windows 10

### Additional Library Versions

```bash
React,
Vue3
```

**Comment by freegithuber** [comment]
是不是你的文件下载目录空间太小了，同样的问题我清理下载目录磁盘空间后解决了。

**Comment by asynch8** [comment]
@freegithuber Translated comment: It's not like your file download directory space is too small, the same problem is solved after I clear the download directory disk space'
(I've google translated the comment and put it in a response here.)

No, I do not believe that is the case, for the following reasons:
- All my disks have much more than 16mb available, at the smallest 3gb, and biggest 4tb available
- I didn't remove any files in between downloading each of the testcases
- It consistently fails on the 'Axios ~16mb' case, while Fetch is working fine for the same file.

**Comment by aymanson** [comment]
Hi @asynch8 , wanna see if you are running chrome under windows when you encounter such issue. Seems recently I also encounter such problem. However, the download is OK under MacOS chrome. So I am still trying to figure out what happened.

**Comment by asynch8** [comment]
@aymanson Correct, I was also running windows, and it does succeed on my OSX machine. 
However, now that I'm testing out the codesandbox on the windows machine again, I do seem to be able to succeed... Not sure what has changed or why it's working now, but it is. 
Guess it might've been something temporary?

**Comment by jhimy-michel** [comment]
@asynch8 did you solve or find out what was the problem? I am facing a similar situation

**Comment by freegithuber** [comment]
> @asynch8 did you solve or find out what was the problem? I am facing a similar situation

我遇到这个问题的时候C盘空间大概只剩2GB，清理C盘空间后就再也没遇到这个问题了。

**Comment by KarimovDev** [comment]
same here. Using fetch instead of Axios didn't solve the issue

**Comment by NilsEngelbach** [comment]
Facing the same problem. Any progress on this issue?

**Comment by NilsEngelbach** [comment]
I was facing a similar issue with "only" 2,5 GB storage left on the primary storage. Increasing it to 5 GB allowed me to downlaod and store a 15 MB Blob in Chrome...

**Comment by asynch8** [comment]
@NilsEngelbach I was actually in a similar situation I believe... I might've cleared out a few gbs on a harddrive which only had a few gbs available at the time, but I'm not sure if it had an immediate effect or not. Didn't connect it as I would assume that 3gb would be enough for a 16mb file.

**Comment by jasonsaayman** [comment]
Thanks for the report and follow-up investigation. The behaviour appears to be Chrome/environment-specific and not consistently reproducible as an Axios issue. Later comments also indicate local disk/storage conditions may be involved.

I am closing this as not reproducible in axios. If it still reproduces on current Chrome and current axios independently of local storage conditions, please open a new issue with a minimal hosted repro.

**Comment by sipa-echo-zaoa** [comment]
I had the same issue while trying to download a 15 mb pdf using axios in chrome with 2.5GB of free space.
Spent several time trying to figure out what the issue was when i found out this github issue.
Tried replicating the same procedure using firefox and indeed the download worked.
